.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file shouldn't be committed.

Reply:
I will make note of this for future commits